README.md

```diff
@@ -190,7 +190,8 @@ or [mino](https://github.com/helm/charts/tree/master/stable/minio#configuration)
 | `app.ingress.enabled`                | Enable install of ingres                         | false                                                   |
 | `app.ingress.host`                   | Hostname to associate ingress with               | uc.ssl-hep.org                                          |
 | `app.ingress.defaultBackend`         | Name of a service to send requests to internal endpoints to | default-http-backend                         |
-| `app.resources`                      | Pass in Kubernetes pod resource spec to deployment to change CPU and memory | { } |               
+| `app.resources`                      | Pass in Kubernetes pod resource spec to deployment to change CPU and memory | { }                          |
+| `app.secret`                         | Name of a Kubernetes Secret which is used to populate environment variables |                              |               
```

Comment:
Since the secrets->Environment Var mapping is hardcoded in the app_deployment, you should start a new section in the README that documents which secrets are supported and what they do